source/guides/clear/mixer.rst

```diff
@@ -760,7 +749,7 @@ content.
      LOCAL_BUNDLE_DIR = "/home/clr/mix/local-bundles"
      LOCAL_REPO_DIR = ""
      LOCAL_RPM_DIR = ""
-     DOCKER_IMAGE_PATH = "clearlinux/mixer"
+     DOCKER_IMAGE_PATH = "clearlinux/mixer" [REPLACEMENT?]
```

Comment:
@bktan8 Will you please confirm that I can completely remove line 752, above? 
My only concern was whether this removal required a REPLACEMENT.

Reply:
@mvincerx - you can replace it with this:

```
#VERSION 1.2

[Builder]
  CERT = "/home/clr/mix/Swupd_Root.pem"
  SERVER_STATE_DIR = "/home/clr/mix/update"
  VERSIONS_PATH = "/home/clr/mix"
  YUM_CONF = "/home/clr/mix/.yum-mix.conf"

[Swupd]
  BUNDLE = "os-core-update"
  CONTENTURL = "<URL where the content will be hosted>"
  VERSIONURL = "<URL where the version of the mix will be hosted>"
  COMPRESSION = ["external-xz"]
  UPSTREAM_BUNDLES_URL = "https://github.com/clearlinux/clr-bundles/archive/"

[Server]
  DEBUG_INFO_BANNED = "true"
  DEBUG_INFO_LIB = "/usr/lib/debug"
  DEBUG_INFO_SRC = "/usr/src/debug"

[Mixer]
  LOCAL_BUNDLE_DIR = "/home/clr/mix/local-bundles"
  LOCAL_REPO_DIR = "/home/clr/mix/local-yum"
  LOCAL_RPM_DIR = "/home/clr/mixx/local-rpms"
  OS_RELEASE_PATH = ""
```

Reply:
@bktan8 See my latest commmit, below.  Please note: 
- Not every `Variable` is explained in the table (This was true prior to this PR)
- In case you want each Variable to be explained, please LMK.

Reply:
@mvincerx - I noticed the `#VERSION` in the new doc is still 1.0, but it should be 1.2.

Reply:
I corrected `#VERSION` to 1.2.